devtools/maven/src/main/java/io/quarkus/maven/QuarkusProjectMojoBase.java

```diff
@@ -165,7 +165,7 @@ private String resolveValue(String expr) throws IOException {
     }
 
     private List<Dependency> getManagedDependencies() throws IOException {
-        final DependencyManagement managed = project.getModel().getDependencyManagement();
+        final DependencyManagement managed = project.getOriginalModel().getDependencyManagement();
```

Comment:
Be careful with that, the caller shouldn't expect that the returned set of dependencies is a complete set. But it looks like it's used in one place only.
In fact, we should be detecting the platforms differently. The current way is not reliable in multi module projects with BOMs imported by parents.

Reply:
something we can fix for 1.6 or a 1.7 thing

Reply:
forgot to add a ? at end of that sentence :) 

@aloubyansky something we can fix for 1.6 or a 1.7 thing ?

Reply:
A somewhat more reliable way would be to be also looking for it in the project.getParent() all the way up until there is no parent. This could easily be done in 1.6.
But then there could be parents outside of the current workspace. Those could be resolved using a resolver. Also doable in a short time frame.
But then the imported BOMs could also be flattened, in which case it all falls apart. That's probably not doable in 1.6 or even 1.7, not sure.

Reply:
The registry needs to know which quarkus core version to use to query for compatible extensions. A workaround may be to use the plugin version since they should always match (at least for now)

Reply:
For that you actually need project.getModel().getDependencyManagement();

Reply:
thanks @aloubyansky - that does sounds like it would even be a problem even today without registry as when we do add/list we rely on locating the locally used platform, right?

Reply:
Right